ext/dcos-installer/dcos_installer/cli.py

```diff
@@ -50,7 +50,15 @@ def print_usage():
 Install Mesosophere's Data Center Operating System
```

Comment:
Not for this PR, but why can't we just use the standard / default generated argparse help nowadays?